test/widgets/recent_dm_conversations_test.dart

```diff
@@ -49,7 +50,7 @@ Future<void> setupPage(WidgetTester tester, {
   await tester.pumpWidget(TestZulipApp(
     accountId: eg.selfAccount.id,
     navigatorObservers: navigatorObserver != null ? [navigatorObserver] : [],
-    child: const RecentDmConversationsPage()));
+    child: const HomePage(initialTab: HomePageTab.directMessages)));
```

Comment:
How about making this just `HomePage()`, and following with a `tester.tap(find.byIcon(…))` to get to the right tab?

That'd be more realistic, and (if we do the same for the subscription/channels tab below) would let us avoid having this `initialTab` field that's used only in tests.